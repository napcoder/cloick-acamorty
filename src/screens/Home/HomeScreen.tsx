import { SafeContainer, Title, TitleContainer } from './HomeScreenStyles'

export default function HomeScreen() {
  return (
    <SafeContainer>
      <TitleContainer>
        <Title>Cloick Acamorty</Title>
      </TitleContainer>
    </SafeContainer>
  )
}
