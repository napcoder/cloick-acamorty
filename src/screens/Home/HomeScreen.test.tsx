import { render, screen } from '@testing-library/react-native'
import React from 'react'
import renderer from 'react-test-renderer'

import HomeScreen from './HomeScreen'

describe('HomeScreen', () => {
  test('Renders snapshot correctly', () => {
    const tree = renderer.create(<HomeScreen />).toJSON()
    expect(tree).toMatchSnapshot()
  })
  describe('Integration', () => {
    it('Should render a title', () => {
      render(<HomeScreen />)

      const title = screen.queryByText('Cloick Acamorty')
      expect(title).toBeTruthy()
    })
  })
})
