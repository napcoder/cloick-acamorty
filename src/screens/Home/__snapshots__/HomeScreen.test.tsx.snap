// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomeScreen Renders snapshot correctly 1`] = `
<RNCSafeAreaView
  edges={
    Array [
      "bottom",
      "left",
      "right",
      "top",
    ]
  }
  style={
    Array [
      Object {
        "alignItems": "stretch",
        "backgroundColor": "rgb(252, 238, 12)",
        "flexBasis": 0,
        "flexGrow": 1,
        "flexShrink": 1,
        "justifyContent": "flex-start",
      },
    ]
  }
>
  <View
    style={
      Array [
        Object {
          "borderRadius": 8,
          "marginHorizontal": 16,
          "marginTop": 16,
        },
      ]
    }
  >
    <Text
      style={
        Array [
          Object {
            "color": "rgb(0, 0, 0)",
            "fontFamily": "ChakraPetch_700Bold",
            "fontSize": 32,
            "fontWeight": "700",
            "textAlign": "center",
          },
        ]
      }
    >
      Cloick Acamorty
    </Text>
  </View>
</RNCSafeAreaView>
`;
